src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="starUnit"></div>
```

Comment:
You have used as kebab-case, as camelCase.
So, please rewrite all classes using ONE case. Here is a link for BEM naming convention [link](https://en.bem.info/methodology/naming-convention/)